backend/routes/door.go

```diff
@@ -11,5 +11,6 @@ func DoorsRoute() chi.Router {
 	r.Post("/", controller.CreateDoor)
 	r.Put("/", controller.UpdateDoor)
 	r.Delete("/{doorId}", controller.DeleteDoor)
+	r.Get("/unlocked/{doorId}", controller.OpenDoor)
```

Comment:
`unlocked` or `unlock`?

Reply:
What does this endpoint do?

Reply:
this is the endpoint cameron is pinging, so it tells him when a door has been "unlocked" by a user so he can run the logic to open it

Reply:
Can you change the controller name this to `IsDoorOpen` or something